src/Reflector/DefaultReflector.php

```diff
@@ -46,7 +46,7 @@ public function reflectClass(string $identifierName): ReflectionClass
      */
     public function reflectAllClasses(): iterable
     {
-        /** @var ReflectionClass[] $allClasses */
+        /** @var list<ReflectionClass> $allClasses */
```

Comment:
Not important, but perhaps one day we should template `IdentifierType` to get inference here